src/core/pages/signin.tsx

```diff
@@ -1,12 +1,25 @@
 import { Theme } from "../.."
 import { InternalProvider } from "../../lib/types"
 
+export type SignInErrorTypes =
```

Comment:
could we add a JSDoc comment on top of this that takes the user to the https://next-auth.js.org/configuration/pages#sign-in-page page, or better yet, each value should have a comment on them. See the link for reference.

Reply:
Thanks for the review! 👍 Just added the `ErrorType` for the error page, also linked in both cases the documentations.